pkg/restapi/backup.go

```diff
@@ -250,6 +252,32 @@ func (h backupHandler) describeSchema(w http.ResponseWriter, r *http.Request) {
 	render.Respond(w, r, convertSchema(cqlSchema))
 }
 
+func (h backupHandler) cleanup(w http.ResponseWriter, r *http.Request) {
+	cluster := mustClusterFromCtx(r)
+
+	var (
+		deleteDiskSnapshots bool
```

Comment:
What is this query parameter defining ? 

If I understand the code correctly, you require to provide the `delete_disk_snapshots=true` every time someone is calling this endpoint.
If the param is missing or is equal to false then you fail the request.

Why ? Something is missing in the implementation ?